v2/internal/app/app_dev.go

```diff
@@ -73,12 +73,18 @@ func CreateApp(appoptions *options.App) (*App, error) {
 	var devServerFlag *string
 	var frontendDevServerURLFlag *string
 	var loglevelFlag *string
+	var obfuscateFlag *bool
 
 	assetdir := os.Getenv("assetdir")
 	if assetdir == "" {
 		assetdirFlag = devFlags.String("assetdir", "", "Directory to serve assets")
 	}
 
+	obfuscate := os.Getenv("WAILS_OBFUSCATE")
+	if obfuscate == "" {
```

Comment:
Could this be a boolean? feels better.

Reply:
Can't. It's an environment variable = string

Reply:
Can it be "1" or "0" as the string values? Or am I showing my GoLang noob status ^.^

Reply:
It can be anything you like. We just check it's set.